features/donations/quick_donation.feature

```diff
@@ -92,3 +97,30 @@ Scenario: admin logged in, records donation on behalf of patron
   And customer "Joe Mallon" should have a donation of $9.00 to "General Fund"
   And an email should be sent to customer "Joe Mallon" containing "$  9.00  Donation to General Fund"
   
+Scenario: landing on quick donation page with valid account code
+  Given I am logged in as customer "Tom Foolery"
```

Comment:
can these Givens be pulled out to a Background block?

Reply:
Done